catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -260,4 +308,22 @@ private QueryRequest createQueryRequest(Filter filter) {
   public WorkspaceMetacardImpl getWorkspaceMetacard(String workspaceId) {
     return getWorkspaceMetacards(Collections.singleton(workspaceId)).get(0);
   }
+
+  @Override
+  public List<Metacard> getAssociationMetacards(String workspaceId) {
+    Query query = new QueryImpl(getAssociationFilter(workspaceId));
+    QueryRequest queryRequest = new QueryRequestImpl(query);
```

Comment:
:pencil2: `QueryRequest queryRequest = new QueryRequestImpl(new QueryImpl(getAssociationFilter(workspaceId)));`

Reply:
I prefer to keep it this way to reduce nesting